### DISCUSSION TITLE
Collaborators skills knowledge

### DISCUSSION THREAD

**Discussion by stafyniaksacha** [strapi-community/.github#5] [Q&A]
Let's discuss how we can know about others collaborator's skills, so we can ask for review to them

**Comment by derrickmehaffy** [comment]
We have this concept of "player profiles" internally at Strapi, basically gives employees a place to talk about themselves and their skill levels, ect. Might be cool to adopt. I'll see if I can share some info about it.

Ideally a basic checklist might help too covering some common topics

**Comment by vcoisne** [comment]
https://github.com/all-contributors/all-contributors might be an interesting option as well.
